src/components/Books.js

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+
+const Books = () => {
+  const { title, author } = useState();
```

Comment:
Please create a `Book` component and render the list of books using this component.

The `Book` component takes the book data as a prop and renders the title and the author. It also has a button for removing books.